automation/models.py

```diff
@@ -45,9 +45,10 @@ def validate_method(cls, v):
 
 
 class VolunteerModel(MetaBaseModel):
```

Comment:
TODO : clean up stragglers to make sure that the volunteer model can parse all prod data